src/hooks/usePlayback.js

```diff
@@ -76,23 +76,12 @@ const usePlayback = (id, onAfterAudioEnd) => {
   }
 
   const loadTrack = useCallback(
-    ({ segments, gateways, title, artistName, mp3StreamUrl }) => {
+    ({ mp3StreamUrl }) => {
       if (!audioRef.current) {
         throw new Error('Init not called')
       }
-      audioRef.current.load(
-        segments,
-        onAudioEnd,
-        [],
-        gateways,
-        {
-          id,
-          title,
-          artist: artistName,
-          artwork: ''
-        },
-        mp3StreamUrl
-      )
+      console.log('load')
```

Comment:
nit: is this (and the `on toggle play` log) staying on purpose?

Reply:
thx! accident